tutorials/baseline/auto_trainer.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
there is no tutorial folder, and also I doubt it should be in baseline folder
I would suggest putting it to examples/trainer folder

Reply:
Ah thanks. I'll have to investigate how that could happen ..